guides/common/assembly_using-external-databases.adoc

```diff
@@ -0,0 +1,37 @@
+[id="using-external-databases_{context}"]
+= Using External Databases with {Project}
+
+ifeval::["{build}" == "foreman"]
+For Red Hat systems only.
+endif::[]
+
+As part of the installation process for {ProjectName}, the *{foreman-installer}* command installs MongoDB and PostgreSQL databases on the same server as {Project}. In certain {Project} deployments, using external databases instead of the default local databases can help with the server load. Depending on your requirements, you can use external databases for either MongoDB or PostgreSQL database, or both.
+
+ifeval::["{build}" == "satellite"]
+Red Hat does not provide support or tools for external database maintenance. This includes backups, upgrades, and database tuning. You must have your own database administrator to support and maintain external databases.
+endif::[]
+
+// Known issue: need to explicitly include the section titles so that two xrefs in the same paragraph render correctly
+Use xref:mongodb-as-an-external-database-considerations_{context}[MongoDB as an External Database Considerations] and xref:postgresql-as-an-external-database-considerations_{context}[PostgreSQL as an External Database Considerations] to decide if you want to use external databases for your {Project} deployment.
+
+[id="external-databases-installation-overview_{context}"]
+== Installation Overview
```

Comment:
Lines 17 and 18 seems redundant. Also, FCC does not allow to use two IDs within one assembly. Removing those two lines will solve it.

Reply:
This is a sub-chapter heading, since the overview isn't a standalone module but does need a heading. It's the same thing as the heading of a module but is just in the text rather than included.

Reply:
The same kind of overview is given in 3.6. Configuring Foreman server with a Custom SSL Certificate, but this time without a special sub-title for it. I think it is fine for it to have no subtitle because the title of the assembly itself is descriptive: 3.7. Using External Databases with Foreman. 
Now when I look at this Installation Overview, I think we can simple remove it, because modules go in the order that users must perform them. A similar list was required in 3.6. Configuring Foreman server with a Custom SSL Certificate because its last step is to complete procedures in another guide. What do you think?

Reply:
I'd rather keep in the overview steps, just because not all of our current assemblies require using all the procedures in order so I'd like to be super clear, but I agree that we can remove the heading. :+1:

Reply:
Added a commit to remove the heading. Looks pretty nice! Let me know if you're happy with this or still want to remove the list.